downloaded/installLisk.sh

```diff
@@ -54,6 +54,15 @@ prereq_checks() {
 		echo "Error: tar is not installed. Exiting."
 		exit 2
 	fi
+
+	PORT_5432_IN_USE=$(netstat -tna | grep -c ':5432 .*LISTEN')
```

Comment:
Does this work on all support Ubuntu LTS releases?

Reply:
Tested and works in 18.04 and 16.04

Reply:
whoops, did not see the new comments

Reply:
I just checked 16.04. Neither fuser nor lsof are installed

Reply:
usually `fuser` comes with the base system with most distros. In ubuntu it's on psmisc package (along with killall and pstree commands). lsof usually doesn't come by default.

Perhaps there is no silver bullet for this and I think this can achieve trying different commands like:

```
#!/bin/bash

if which fuser ; then
    CHECKCMD=$(fuser -n tcp 5432)
elif which lsof ; then
    CHECKCMD=$(lsof example)
elif which ss ; then
    CHECKCMD=$(ss example)
else ;
   CHECKCMD=$(netstat example)
fi
```

Reply:
Trying to use `ss` and falling back to `netstat` should be enough to support all linux distros and macos.